### DISCUSSION TITLE
Question: does the "console_title_template" variable set the "Tab Title" with Windows Terminal

### DISCUSSION THREAD

**Discussion by Daksol** [JanDeDobbeleer/oh-my-posh#4845] [Troubleshoot]
I am using Windows Terminal with Powershell 7, and have an OhMyPosh prompt working well.

In the WT settings for Powershell I have configured "tabTitle" as "PowerShell7 Core". That title displays briefly, but then reverts to "pwsh".

Is the OhMyPosh variable "console_title_template" used to set the tab title in Windows Terminal ?

Thanks

**Comment by JanDeDobbeleer** [comment]
Yes. And I guess that's the default.
